### PR TITLE
Issue #2707469 media browser design

### PR DESCRIPTION
**Pull request by Sam152**

### CODE REVIEW

**Comment by Sam152** [comment]
To work on this:

* Check out this branch
* Do a "phing build:dev" to download and install everything.
* Delete agov/modules/contrib/media_entity_browser
* Clone from Drupal.org to the same place.
* Commit any improvements to the contrib module & push.
* Update the commit in the make file to be the latest commit in the contrib repo.

**Comment by Sam152** [comment]
@rikki-iki Steps to work on this above. Basically open season to change CSS/JS/view config to make this look good :)

**Comment by rikki-iki** [comment]
screen shot 2016-04-20 at 7 22 29 pm

File: agov/modules/custom/agov_media/config/install/editor.editor.text_and_media.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ settings:
           name: Media
           items:
             - Blockquote
-            - embed
```
Comment (benjy) [inline agov/modules/custom/agov_media/config/install/editor.editor.text_and_media.yml]: Do we inherit this dependency from media_embed now?

**Comment by benjy** [comment]
LGTM

**Pull request merged**